src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/VilkårTabellRadEndre.tsx

```diff
@@ -211,6 +211,19 @@ const VilkårTabellRadEndre: React.FC<IProps> = ({
         }
     };
 
+    const erBegrunnelsePåkrevd = (): boolean => {
+        if (
```

Comment:
Kan dette gjøres direkte?
```suggestion
const erBegrunnelsePåkrevd = (): boolean => 
        (toggles[ToggleNavn.skjønnsvurdering] && redigerbartVilkår.verdi.erSkjønnsmessigVurdert) ||
        (toggles[ToggleNavn.medlemskap] && redigerbartVilkår.verdi.erMedlemskapVurdert)
```